packages/backend/src/services/PromoteService/PromoteService.ts

```diff
@@ -204,6 +204,7 @@ export class PromoteService extends BaseService {
                     subject('Space', {
                         organizationUuid,
                         projectUuid: upstreamContent.projectUuid,
+                        isPrivate: upstreamContent.space.isPrivate,
```

Comment:
Fixes a bug for public spaces (added new profileAbility) 

Before
![Screenshot from 2024-11-28 10-14-20](https://github.com/user-attachments/assets/2e7b9f1b-bb4c-41a6-bbd8-58f4adfca590)

After

![Screenshot from 2024-11-28 10-23-50](https://github.com/user-attachments/assets/676a71e6-4fd5-4866-b9eb-9332095025da)